src/main/java/com/rocketden/main/dto/problem/ProblemDto.java

```diff
@@ -0,0 +1,11 @@
+package com.rocketden.main.dto.problem;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ProblemDto {
+  private String name;
+  private String description;
```

Comment:
Is this where we will add the fields indicating problem difficulty and type too?

Reply:
Yeah that's right! Any of the problem details; name, description, example I/O, and maybe test cases (or at least their IDs) would go here. I kept it simple for now because we aren't going to be using all that for a while. 👍